package.json

```diff
@@ -25,7 +26,9 @@
   },
   "homepage": "https://github.com/thebigredgeek/apollo-errors#readme",
   "dependencies": {
-    "es6-error": "^4.0.0"
+    "assert": "^1.4.1",
+    "extendable-error": "^0.1.5",
+    "install": "^0.10.1"
```

Comment:
What's this?

Reply:
`"extendable-error": "^0.1.5"`is required for parent class of ApolloError, so it typescript version of es6-error
`"assert": "^1.4.1"` is required for asserting types in typescript

Have removed `install`